components/x-gift-article/src/HighlightRecipientMessage.scss

```diff
@@ -0,0 +1,66 @@
+@import '@financial-times/o-colors/main';
+@import '@financial-times/o-spacing/main';
+
+.highlight-recipient-message {
+  .highlight-message-content {
+    position: relative;;
+    background-color: oColorsByName('slate');
+    color: oColorsMix('ft-grey', 'white', 20);
+    padding: oSpacingByName('s3');
+  
+    .title {
+      @include oTypographySans($scale: 1, $weight: 'regular');
+      margin: 0;
+      color: oColorsByName('white');
+    }
+  
+    .description {
+      @include oTypographySans($scale: 0);
+      margin: 0;
+    }
+    
+    .divider{
+      padding-top: oSpacingByName('s2');
+      padding-bottom: oSpacingByName('s3');
+      &::after {
+        border-top: oSpacingByName('s1') solid oColorsByName('mint');
+        content: "";
+        position: absolute;
+        width: 40px;
+        content: "";
+        display: block;
+      }
+    }
+  
+    .close {
+      background-image: url('https://www.ft.com/__origami/service/image/v2/images/raw/fticon-v1%3Across?source=editor&tint=%23FFFFFF&bgcolor=%23FFFFFF');
+      background-repeat: no-repeat;
+      background-size: contain;
+      background-position: 50%;
+      background-color: transparent;
+      width: oSpacingByName('s6');
+      height: oSpacingByName('s6');
+      border: none;
+      padding: 0;
+      content: "";
+      position: absolute;
+      right: oSpacingByName('s2');
+      top: oSpacingByName('s2');
+    }
+  
+    .save-highlights-button {
+      color: oColorsByName('mint');
+      background: none;
+      border: none;
+      text-decoration: underline;
+
+      &:hover {
+        color: oColorsByName('mint-70');
+        border-bottom: oColorsByName('mint-70');
+        text-decoration-color: oColorsByName('mint-70');
+      }
```

Comment:
Since `save-highlights-button` looks like an inverse professional link, I can just use the `o-typography` stuff to style it instead of `o-buttons`,  because there is a lot of style difference between the buttons and the links.